Source/OrchBase/OrchBaseLink.cpp

```diff
@@ -161,6 +161,13 @@ for(unsigned i=0;i<lim*4;i++) {
 
 //------------------------------------------------------------------------------
 
+/* MLV commented this out on 2020-03-31 because he couldn't find any
+ * documentation on it (so he couldn't understand what it was supposed to do),
+ * and because it broke compilation when removing the now-unneccesary `Bin`
+ * class. */

```

Comment:
Let's leave it commented for now, and we can resolve it later when we define this logic properly. It might be useful, and it doesn't cost much to leave it there.